wandb/internal/sender.py

```diff
@@ -523,7 +523,7 @@ def send_artifact(self, data):
         saver.save(
             type=artifact.type,
             name=artifact.name,
-            metadata=json.loads(artifact.metadata),
+            metadata=json.loads(artifact.metadata or "{}"),
```

Comment:
I found this one when I ran `wandb artifact put -n artifact_name filename.txt`.

Reply:
I think I broke this in https://github.com/wandb/client-ng/pull/197. My bad, thanks for finding and fixing it!